astropy/io/misc/asdf/tags/time/tests/test_timedelta.py

```diff
@@ -0,0 +1,20 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+import pytest
+
+asdf = pytest.importorskip('asdf')
+from asdf.tests.helpers import assert_roundtrip_tree
+
+from astropy.time import Time
+
+
+@pytest.mark.parametrize('fmt', Time.FORMATS.keys())
```

Comment:
Maybe have a separate test where we iterate over all scales (and include `None`).